### PR TITLE
Update with the Polymer 1.1 style recommendations

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

**Comment by abdonrd** [comment]
Any news about this PR? :see_no_evil:

It has already three months. :confused:

File: bower.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "paper-behaviors": "PolymerElements/paper-behaviors#^1.0.0",
     "paper-icon-button": "PolymerElements/paper-icon-button#^1.0.0",
     "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "polymer": "Polymer/polymer#^1.0.0"
+    "polymer": "Polymer/polymer#^1.1.0"
```
Comment (bicknellr) [inline bower.json]: This update works just as well in 1.0.0, right?

Comment (abdonrd) [inline bower.json]: No, I changed to [new style recommendations of Polymer 1.1](https://blog.polymer-project.org/announcements/2015/08/13/1.1-release/).

Comment (bicknellr) [inline bower.json]: Oh! I thought this was supported before.

Comment (abdonrd) [inline bower.json]: Move the `<style>` tag inside the `<template>` tag is only from 1.1. :blush:

[1.1 Release](https://blog.polymer-project.org/announcements/2015/08/13/1.1-release/):
*Previously, we recommended that a `<style>` element should be placed inside an element’s `<dom-module>` but outside of its template. This is still supported, but we’ve now optimized placing styles within the template itself, so having the `<style>` tag outside of the template will be slower. We will be updating all of the elements provided by the Polymer team to match this new recommended behavior as well.*

**Comment by abdonrd** [comment]
Anything else, @bicknellr? :)

**Comment by bicknellr** [comment]
Nope, looks good! Thanks for the clarification.

**Pull request merged**